reference/docs-conceptual/whats-new/differences-from-windows-powershell.md

```diff
@@ -474,8 +609,148 @@ For example, the following two strings are considered to be identical:
+script errors with a new default view **ConciseView**. The views are user-selectable through the
+preference variable `$ErrorView`.
+
+With **ConciseView**, if an error is not from a script or parser error, then it's a single line
+error message:
+
+```powershell
+Get-Childitem -Path c:\NotReal
+```
+
+```Output
+Get-ChildItem: Cannot find path 'C:\NotReal' because it does not exist
+```
+
+If the error occurs during script execution or is a parsing error, PowerShell returns a multiline
+error message that contains the error, a pointer and error message showing where the error is in
+that line. If the terminal doesn't support ANSI color escape sequences (VT100), then colors are not
+displayed.
+
+The default view in PowerShell 7 is **ConciseView**. The previous default view was **NormalView**
+and you can select this by setting the preference variable `$ErrorView`.
+
+```powershell
+$ErrorView = 'NormalView' # Sets the error view to NormalView
+$ErrorView = 'ConciseView' # Sets the error view to ConciseView
+```
+
+> [!NOTE]
+> A new property **ErrorAccentColor** is added to `$Host.PrivateData` to support changing
+> the accent color of the error message.
+
+A new cmdlet `Get-Error` provides a complete detailed view of the fully qualified error when
+desired. By default the cmdlet displays the full details, including inner exceptions, of the last
+error that occurred.
+
+The `Get-Error` cmdlet supports input from the pipeline using the built-in variable `$Error`.
+`Get-Error` displays all piped errors.
+
+```powershell
+$Error | Get-Error
+```
+
+The `Get-Error` cmdlet supports the **Newest** parameter, allowing you to specify how many errors
+from the current session you wish displayed.
+
+```powershell
+Get-Error -Newest 3 # Displays the lst three errors that occurred in the session
+```
+
+For more information about [Get-Error][Get-Error].
+
 ## Cmdlet changes
 
+### Parallel execution added to ForEach-Object
+
+Beginning in PowerShell 7.0, the `ForEach-Object` cmdlet, which iterates items in a collection, now
+has built-in parallelism with the new **Parallel** parameter.
+
+By default, parallel script blocks use the current working directory of the caller that started the
+parallel tasks.
+
+This example retrieves 50,000 log entries from 5 system logs on a local Windows machine:
+
+```powershell
+$logNames = 'Security','Application','System','Windows PowerShell','Microsoft-Windows-Store/Operational'
+
+$logEntries = $logNames | ForEach-Object -Parallel {
+    Get-WinEvent -LogName $_ -MaxEvents 10000
+} -ThrottleLimit 5
+
+$logEntries.Count
+
+50000
+```
+
+The **Parallel** parameter specifies the script block that is run in parallel for each input log
+name.
+
+The new **ThrottleLimit** parameter limits the number of script blocks running in parallel at a
+given time. The default is 5.
+
+Use the `$_` variable to represent the current input object in the script block. Use the
```

Comment:
Does it _have_ to be `$_` or can a user use `$PSItem`?

Reply:
It can be either but we always use `$_` rather than `$PSItem` in our docs.